spec/integration_spec.rb

```diff
@@ -1010,25 +1010,27 @@ class SerializedDocument < ActiveRecord::Base
     Book.index(safe_index_uid('Book')).delete_all_documents
   end
 
+  # rubocop:disable RSpec/MultipleExpectations
```

Comment:
I think we could disable this spec for the entire suite because pretty much all of the tests have one or more assertions.

What do you think about this?